dialogue-core/src/test/java/com/palantir/dialogue/core/RetryingChannelTest.java

```diff
@@ -55,140 +55,144 @@
     private static final Request REQUEST = Request.builder().build();
 
     @Mock
-    private Channel channel;
+    private EndpointChannel channel;
 
     @Test
     public void testNoFailures() throws ExecutionException, InterruptedException {
-        when(channel.execute(any(), any())).thenReturn(SUCCESS);
+        when(channel.execute(any())).thenReturn(SUCCESS);
```

Comment:
assume the test changes are just refactors, so i'm just skimming

Reply:
yup, didn't add/remove any of em.